examples/python/ex3-volume.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and other CEED contributors.
+# All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This file is part of CEED:  http://github.com/ceed
+
+"""
+Example using libCEED to compute the volume of a 3D body using matrix-free application of a mass operator.
+This example also uses a diffusion operator, which provides zero contribution to the computed volume but
+demonstrates libCEED's ability to handle multiple basis evaluation modes for the same input and output vectors.
+The example supports arbitrary mesh and solution degrees in 1D, 2D, and 3D from the same code.
+"""
+
+import argparse
+import math
+import numpy as np
+import libceed
+from libceed import GAUSS, VECTOR_ACTIVE, VECTOR_NONE, ELEMRESTRICTION_NONE, BASIS_NONE
+from libceed import EVAL_INTERP, EVAL_GRAD, EVAL_WEIGHT, EVAL_NONE
+from libceed import MEM_HOST, COPY_VALUES, USE_POINTER
+import os
+from sysconfig import get_config_var
+import ctypes
+
+
+def load_qfs_so():
+    """Load the QFunctions shared library.
+
+    Returns:
+        Loaded shared library object
+    """
+    file_dir = os.path.dirname(os.path.abspath(__file__))
+    qfs_so = os.path.join(
+        file_dir,
+        "libceed_qfunctions" + get_config_var("EXT_SUFFIX"))
+
+    # Load library
+    return ctypes.cdll.LoadLibrary(qfs_so)
```

Comment:
```suggestion
def find_qfs_so(name, path):
    """Find the QFunctions shared library.

    Returns:
        Filepath to shared library object
    """
    for root, dirs, files in os.walk(path):
        if name in files:
            return os.path.join(root, name)


def load_qfs_so():
    """Load the QFunctions shared library.

    Returns:
        Loaded shared library object
    """
    file_dir = os.path.join(
        os.path.dirname(os.path.abspath(__file__)),
        "build")
    qfs_so = find_qfs_so(
        "libceed_qfunctions" + get_config_var("EXT_SUFFIX"),
        file_dir)

    # Load library
    return ctypes.cdll.LoadLibrary(qfs_so)
```
I had trouble loading the .so on my system. This should fix that